<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_classxinxi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>课程信息</string>
   </property>
   <widget class="QLabel" name="label_classtime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>上课时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_classname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>课程名称：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_teacher">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>任课教师：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_classname">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>54</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_teacher">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>84</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_classtime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>出勤汇总</string>
   </property>
   <widget class="QLabel" name="label_class">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>班      级：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>班级人数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_class">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>19</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_stunum">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_numture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>58</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>实到人数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_stunumtrue">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>返回主窗口</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>211</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>将信息保存至SD卡或者U盘文本文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
